### PR TITLE
8.0 rename selection remark

### PR DESCRIPTION
**Pull request by yvaucher**
Depends on
- [x] https://github.com/OCA/purchase-workflow/pull/111

### CODE REVIEW

**Comment by gurneyalex** [comment]
:+1:

**Comment by lepistone** [comment]
Thanks :+1:

**Comment by lepistone** [comment]
I temporarily pulled the dependency OCA/purchase-workflow#111 so we can merge them together.

**Comment by lepistone** [comment]
with the temporary pull of OCA/purchase-workflow#111, travis is green (and runbot is red because the pending branch is on travis only).

I am deleting the temporary commit, I think we can merge then (cc @gurneyalex).
Thanks!

**Comment by lepistone** [comment]
removal of temporary commit: done.
I kill the travis build because it is useless now.

**Comment by lepistone** [comment]
hello and happy easter to all :rabbit: :rabbit2: :egg: 

TL;DR this should be merged as the same time as OCA/purchase-workflow#111, and the build with the two together is green. I already removed the temporary commit that pulled that dependency into the travis build, so we're ready to merge the two at the same time with merge buttons when someone approves this one.

Thanks!

**Comment by jgrandguillaume** [comment]
:+1:
